### PR TITLE
Pin dependencies in docker file.

### PR DESCRIPTION
**Pull request by godofredoc**
The docker file had two dependencies that were not locked. An npm
dependency and the debian base image. This PR pins those two
dependencies.

## Pre-launch Checklist

- [X] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [X] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [X] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [X] I signed the [CLA].
- [X] I listed at least one issue that this PR fixes in the description above.
- [X] I updated/added relevant documentation (doc comments with `///`).
- [X] I added new tests to check the change I am making, or this PR is [test-exempt].
- [X] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

**Comment by christopherfujino** [comment]
I don't think pinning the image to an exact sha makes much difference since we're running an `apt-get update && apt-get upgrade` right after. Also, I think that pinning exact OS dependencies without also autorolling them would be a net negative security wise, as we wouldn't get routine security backports, for example to openSSL.

**Comment by godofredoc** [comment]
> I don't think pinning the image to an exact sha makes much difference since we're running an `apt-get update && apt-get upgrade` right after. Also, I think that pinning exact OS dependencies without also autorolling them would be a net negative security wise, as we wouldn't get routine security backports, for example to openSSL.

A follow up PR will enable dependabot for Dockerfiles. Ideally dependabot will be the roller for the dependencies and the updates will force to regenerate the containers with the latest changes.

**Comment by christopherfujino** [comment]
> > I don't think pinning the image to an exact sha makes much difference since we're running an `apt-get update && apt-get upgrade` right after. Also, I think that pinning exact OS dependencies without also autorolling them would be a net negative security wise, as we wouldn't get routine security backports, for example to openSSL.
> 
> A follow up PR will enable dependabot for Dockerfiles. Ideally dependabot will be the roller for the dependencies and the updates will force to regenerate the containers with the latest changes.

nice

File: .cirrus.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ gcp_credentials: ENCRYPTED[!cc769765170bebc37e0556e2da5915ca64ee37f4ec8c966ec147
 task:
   gke_container:
     dockerfile: "dev/ci/docker_linux/Dockerfile"
-    builder_image_name: docker-builder # gce vm image
+    builder_image_name: docker-builder-linux # gce vm image
```
Comment (christopherfujino) [inline .cirrus.yml:21]: was this changed in the GCE config?

Comment (godofredoc) [reply .cirrus.yml:21]: Yes, we created a new images and configurations to increase the disk size to 100GB the previous image with 10GB was running out of space.

Comment (christopherfujino) [reply .cirrus.yml:21]: sounds good

**Review by christopherfujino — APPROVED** [review]
LGTM

**Pull request merged**